geode-core/src/main/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImpl.java

```diff
@@ -249,8 +249,8 @@ protected void updateLocatorInLocatorList(HostAddress locator) {
         }
       }
 
-      logger.info("updateLocatorInLocatorList locator list from: {} to {}",
-          locatorList.getLocators(), newLocatorsList);
+      // logger.debug("updateLocatorInLocatorList locator list from: {} to {}",
```

Comment:
Delete the commented out lines.

Reply:
I actually meant to put that line back...